dataset_loaders/images/mnistm.py

```diff
@@ -0,0 +1,183 @@
+"""Mnist-m dataset."""
+import numpy as np
+import os
+import time
+from PIL import Image
+from images.mnist import MnistDataset
+
+
+class MnistMDataset(MnistDataset):
+    """The mnist-m handwritten digit dataset.
+
+    The dataset should be downloaded from [1]
```

Comment:
Will do. I want to add the download to mnistm via my website akanev.com. Should I wait to push changes until I link it, or can I push a change to just the mnistm download link later on?

Reply:
Will push changes now and then push that later if you choose to accept it :)

Reply:
Is there any advantage to download through your website rather than the original download link? I think it's best to link to an established website to avoid to potentially generate confusion in the users and to make sure the link is up to date in case the original authors notice some problem with the data.

Reply:
There is no really established website for mnistm. I believe Yaroslav, (a MILA phd student), created it for previous research. There is one link to the old mnistm data via google drive, which I think could look sketchy. The new mnistm that I generated using a script is not available via the web right now, which is why I wanted to upload it.

Reply:
Also, theres some errors in the code when inheriting mnistm from mnist that I need to work out. So I will just push mnist for now, and fix the mnistm issues in the meantime.